schema/DoctrineMigrations/Version20220308132214.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220308132214 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Generic Voicemail support';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE Voicemails (
+              id INT UNSIGNED AUTO_INCREMENT NOT NULL,
+              enabled TINYINT(1) UNSIGNED DEFAULT \'1\' NOT NULL,
+              userId INT UNSIGNED DEFAULT NULL,
+              residentialDeviceId INT UNSIGNED DEFAULT NULL,
+              companyId INT UNSIGNED NOT NULL,
+              name VARCHAR(200) NOT NULL,
+              email VARCHAR(200) DEFAULT NULL,
+              sendMail TINYINT(1) UNSIGNED DEFAULT \'0\' NOT NULL,
+              attachSound TINYINT(1) UNSIGNED DEFAULT \'1\' NOT NULL,
+              locutionId INT UNSIGNED DEFAULT NULL,
+          UNIQUE INDEX UNIQ_5CE3741564B64DCC (userId),
```

Comment:
There is no index definition by human beings in the mappings, they were generated by doctrine, due to a one-to-one relationship

Reply:
Actually, you can:

`   <mapped-superclass name="Ivoz\Provider\Domain\Model\Voicemail\VoicemailAbstract" table="voicemail_abstract">
+    <unique-constraints>
+      <unique-constraint name="voicemail_uniqueUserId" columns="userId"/>
+    </unique-constraints>
`
will generate:
`# rm -fr var/cache/* && bin/console doctrine:schema:update --dump-sql
 The following SQL statements will be executed:

     ALTER TABLE Voicemails RENAME INDEX uniq_5ce3741564b64dcc TO voicemail_uniqueUserId;
`
We've many missing named unique indexes so this could go on it's own separate issues as well